tests/foreman/ui/test_activationkey.py

```diff
@@ -1109,36 +1097,38 @@ def test_positive_service_level_subscription_with_custom_product(session):
     """
     org = entities.Organization().create()
     manifests.upload_manifest_locked(org.id)
-    entities_ids = setup_org_for_a_custom_repo({'url': FAKE_1_YUM_REPO, 'organization-id': org.id})
+    entities_ids = setup_org_for_a_custom_repo(
+        {'url': constants.repos.FAKE_1_YUM_REPO, 'organization-id': org.id}
+    )
     product = entities.Product(id=entities_ids['product-id']).read()
     activation_key = entities.ActivationKey(id=entities_ids['activationkey-id']).read()
     # add the default RH subscription
     subscription = entities.Subscription(organization=org).search(
-        query={'search': f'name="{DEFAULT_SUBSCRIPTION_NAME}"'}
+        query={'search': f'name="{constants.DEFAULT_SUBSCRIPTION_NAME}"'}
     )[0]
     activation_key.add_subscriptions(data={'quantity': 1, 'subscription_id': subscription.id})
     # ensure all the needed subscriptions are attached to activation key
     results = activation_key.subscriptions()['results']
-    assert {product.name, DEFAULT_SUBSCRIPTION_NAME} == {
+    assert {product.name, constants.DEFAULT_SUBSCRIPTION_NAME} == {
         ak_subscription['name'] for ak_subscription in results
     }
     # Set the activation service_level to Premium
     activation_key.service_level = 'Premium'
     activation_key = activation_key.update(['service_level'])
-    with VirtualMachine() as vm:
-        vm.install_katello_ca()
-        vm.register_contenthost(org.label, activation_key=activation_key.name)
-        assert vm.subscribed
-        result = vm.run('subscription-manager list --consumed')
-        assert result.return_code == 0
-        assert f'Subscription Name:   {product.name}' in '\n'.join(result.stdout)
-        with session:
-            session.organization.select(org.name)
-            chost = session.contenthost.read(vm.hostname, widget_names='subscriptions')
-            subscriptions = {
-                subs['Repository Name'] for subs in chost['subscriptions']['resources']['assigned']
-            }
-            assert product.name in subscriptions
+
+    rhel7_contenthost.install_katello_ca()
+    rhel7_contenthost.register_contenthost(org.label, activation_key=activation_key.name)
+    assert rhel7_contenthost.subscribed
+    result = rhel7_contenthost.run('subscription-manager list --consumed')
+    assert result.return_code == 0
```

Comment:
I believe this should check status, `return_code` doesn’t exist 
```suggestion
    assert result.status == 0
```